apps/chat_server/lib/chat_server/schemas/user.ex

```diff
@@ -11,8 +11,9 @@ defmodule ChatServer.Schema.User do
   schema "users" do
     field :name, :string
     field :type, :string, default: @default_type
-
     timestamps()
+
+    many_to_many :messages, ChatServer.Schema.Message, join_through: "starred_messages"
```

Comment:
Same as above, `messages` feels like a too generic name. Maybe `starred_messages`?

Reply:
ah okay, I didn't make the connection that I have the freedom to have the name differ from the schema here. `starred_messages` sounds great. The name is a little harder to figure out on the Message side, maybe `users_following`?